.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,28 @@
+name: Question
+description: Ask a question about using Custodian
+labels: [kind/question]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to ask a question!
+  - type: textarea
+    id: question
+    attributes:
+      label: Ask your question
+      description: A clear and concise description of what your trying to do, as well as the underlying use case.
```

Comment:
`A clear and concise description of what your trying to do, as well as the underlying use case.` --> `A clear and concise description of what you are trying to do and the underlying use case.`